<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YACHU!!!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Dice!</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>181</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>16</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>34</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>One</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Two</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Three</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>Four</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Five</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>Six</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>Total</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>Bonus</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>3ofKind</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>4ofKind</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string>FullHouse</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string>S.Strate</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string>L.Strate</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string>Chance</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string>YaChu</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string>TOTAL</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ClickLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>enamy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="text">
     <string>YACHU LIST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>YOU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ENAMY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Single Play</string>
    </property>
    <addaction name="actionVS_Computer"/>
    <addaction name="separator"/>
    <addaction name="actionOne_Man_Play"/>
   </widget>
   <widget class="QMenu" name="menuasdasdasd">
    <property name="title">
     <string>Multi Play</string>
    </property>
    <addaction name="actionCreate_Game"/>
    <addaction name="separator"/>
    <addaction name="actionJoin_Game"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuasdasdasd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_Game">
   <property name="text">
    <string>Create Game</string>
   </property>
  </action>
  <action name="actionJoin_Game">
   <property name="text">
    <string>Join Game</string>
   </property>
  </action>
  <action name="actionVS_Computer">
   <property name="text">
    <string>VS Computer</string>
   </property>
  </action>
  <action name="actionOne_Man_Play">
   <property name="text">
    <string>One-Man Play</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>yachupkg/MyLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YACHU!!!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Dice!</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>181</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>16</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>34</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>One</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Two</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Three</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>Four</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Five</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>Six</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>Total</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>Bonus</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled</set>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>3ofKind</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>4ofKind</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string>FullHouse</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string>S.Strate</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string>L.Strate</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string>Chance</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string>YaChu</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string>TOTAL</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ClickLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>dice1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>enamy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="text">
     <string>YACHU LIST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>YOU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ENAMY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Single Play</string>
    </property>
    <addaction name="actionVS_Computer"/>
    <addaction name="separator"/>
    <addaction name="actionOne_Man_Play"/>
   </widget>
   <widget class="QMenu" name="menuasdasdasd">
    <property name="title">
     <string>Multi Play</string>
    </property>
    <addaction name="actionCreate_Game"/>
    <addaction name="separator"/>
    <addaction name="actionJoin_Game"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuasdasdasd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_Game">
   <property name="text">
    <string>Create Game</string>
   </property>
  </action>
  <action name="actionJoin_Game">
   <property name="text">
    <string>Join Game</string>
   </property>
  </action>
  <action name="actionVS_Computer">
   <property name="text">
    <string>VS Computer</string>
   </property>
  </action>
  <action name="actionOne_Man_Play">
   <property name="text">
    <string>One-Man Play</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>yachupkg/MyLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
>>>>>>> 983daa3260d844f59db8b29f834314c2b87a4211
